.github/workflows/pull_request.yml

```diff
@@ -60,14 +60,3 @@ jobs:
           src:
             - './.github/workflows/image_base.yml'
             - './build/Dockerfile.builder'
-
-  image_base:
```

Comment:
@SamYuan1990 I removed this due to the CI error [here](https://github.com/sustainable-computing-io/kepler/actions/runs/8054337624)

Reply:
https://github.com/sustainable-computing-io/kepler/pull/1262#pullrequestreview-1903252884 yes... @rootfs you had disabled workflow call for the job. hence we need to add workflow call back.

Reply:
we need to revert or roll back https://github.com/sustainable-computing-io/kepler/pull/1262 as this breaks lots of CI pipelines.

Reply:
btw, #1262 https://github.com/sustainable-computing-io/kepler/actions/runs/8051024777 PR merged with CI failure?

Reply:
@rootfs , please check https://github.com/sustainable-computing-io/kepler/pull/1267

Reply:
that makes sense, i merged 1267